source/common/router/config_impl.h

```diff
@@ -457,6 +457,56 @@ class RouteTracingImpl : public RouteTracing {
   Tracing::CustomTagMap custom_tags_;
 };
 
+/**
+ * Implementation of PathMatchPolicy that reads from the proto
+ * PathMatchPolicy of the RouteAction.
+ */
+class PathMatchPolicyImpl : public PathMatchPolicy {
+public:
+  PathMatchPolicyImpl(const envoy::config::core::v3::TypedExtensionConfig typed_config,
+                      ProtobufMessage::ValidationVisitor& validator, std::string path);
+
+  // Default constructor that disables internal redirect.
+  PathMatchPolicyImpl();
+
+  bool enabled() const override { return enabled_; }
```

Comment:
nit: our style is to collect methods that implements of interfaces in (e.g. in envoy/router/path_match_policy.h) together and place a comment above them with the interface name. So, `// Router::PathMatchPredicate` above enabled and predicate.

Reply:
changed throughout